test/Validate.Capability.cpp

```diff
@@ -127,7 +127,9 @@ const vector<string>& AllCapabilities() {
     "StorageImageReadWithoutFormat",
     "StorageImageWriteWithoutFormat",
     "MultiViewport",
-    "SubgroupDispatch"};
+    "SubgroupDispatch",
+    "NamedBarrier",
```

Comment:
Turns out that was already done on master via other PRs.  This diff goes away after rebasing.